### ISSUE TITLE
Gantt chart - multiple scroll bars are getting generated for multiple y-axis

### ISSUE THREAD

**Issue by tejaspchordiya** [highcharts/highcharts#10641]
#### Expected behaviour
Gantt chart - multiple scroll bars are getting generated for multiple y-axis.

#### Actual behaviour
Gantt chart - multiple scroll bars should not be generated for multiple y-axis.

#### Live demo with steps to reproduce
[Demo](https://jsfiddle.net/d5hm79a8/)

**Comment by pawelfus** [comment]
Thanks for reporting @tejaspchordiya 

At this moment, it's possible to sync axes to one scrollbar, defined in first grid column: https://jsfiddle.net/BlackLabel/6p8c0dkt/

Snippet:
```js
  yAxis: {
    // Let all grids render all half-categories too
    startOnTick: false,
    endOnTick: false,
    grid: {
      columns: [{
        // Enable scrollbar:
        scrollbar: {
          enabled: true
        },
        // Sync axes:
        events: {
          afterSetExtremes: function(e) {
            var selfAxis = this;

            selfAxis.chart.yAxis.forEach(function(axis) {
              if (axis !== selfAxis) {
                axis.setExtremes(e.min, e.max, false);
              }
            });

            selfAxis.chart.redraw(false);
          }
        }
      }, { ... }, { ... }]
    }
  }
```

**Comment by tejaspchordiya** [comment]
Thanks for the workaround. But, I think this functionality should be provided Highcharts gantt library OOTB, without any work around.

**Comment by karolkolodziej** [comment]
This issue was fixed by #13952 and changes are available at v.8.2.0.
